scripts/dep_uninstaller.sh

```diff
@@ -10,6 +10,9 @@ if [ -d "/usr/local/include/eosio" ] || [ -d "$HOME/opt/eosio" ] || [[ $1 =~ for
                if [ $( uname ) == "Darwin" ]; then
                   # gettext and other brew packages are not modified as they can be dependencies for things other than eosio
                   brew uninstall llvm@4 --force
+                  brew uninstall doxygen --force
```

Comment:
These additions seem somewhat against the original comment. Force removing could break other packages the user has installed right?

Reply:
I can remove: does that mean we should avoid it for llvm@4 as well?

Reply:
the risk with llvm@4 seems very low since I assume very few things in brew depend on llvm@4. But the new additions are pretty easy to imagine being something the user needs directly or indirectly. We have no idea if eosio installed those or not, nor any idea if the user is using them for something since building eosio. I think we should remove at least the new ones for now

Reply:
Added noninteractive support to the uninstaller script as well prompts to uninstall them. I personally save a lot of time having them available while I'm testing changes locally on my mac.